### PR TITLE
[MWPW-152181] Tabbing gets "stuck" after second instance of merch-card CTAs, within a segment blade

### PR DESCRIPTION
**Pull request by rohitsahu**
Allow the default tab action to proceed if the current focused element is from the last merch card. npm run build changes.

Fix #[MWPW-152181](https://jira.corp.adobe.com/browse/MWPW-152181)

Milo Test link:
https://mwpw-152181--milo--rohitsahu.hlx.page/drafts/rosahu/rosahu

URL for testing:
https://mwpw-152181--mas--adobecom.hlx.page/

Test URLs:
Before: https://main--milo--adobecom.hlx.page/drafts/rosahu/rosahu
After: https://mwpw-152181--milo--rohitsahu.hlx.page/drafts/rosahu/rosahu

### CODE REVIEW

File: web-components/src/merch-card.js [hunk 1]
```diff
@@ -612,6 +612,8 @@ export class MerchCard extends LitElement {
                 !e.shiftKey &&
                 document.activeElement === lastFocusableElement
             ) {
+                const isLastMerchCard = this === document.querySelectorAll(MERCH_CARD_NODE_NAME).item(document.querySelectorAll(MERCH_CARD_NODE_NAME).length - 1);
```
Comment (npeltier) [inline web-components/src/merch-card.js:615]: - do you have lint enabled on your vs code? this line looks too long :) 
- this looks suboptimal and non functional : PR title says "within a segment blade" but you are searching over the whole document. Can you have a look at a page test with different containers (like 2 segment blades) and see how it works? also you should be doing querySelectorAll only once.

**Review by npeltier — CHANGES_REQUESTED** [review]

File: web-components/src/merch-card.js (hunk 1 above)
Comment (rohitsahu) [reply web-components/src/merch-card.js:615]: I think we have to do it for the last card only, we can navigate between the card segments using up and down arrows but it is the last card where the user can stuck.
Other points, I have taken care.

**Comment by rohitsahu** [comment]
@npeltier there are auto changes inside libs folder as well after I ran "npm run build". Should I take those or discard from the PR?

File: web-components/src/merch-card.js [hunk 2]
```diff
@@ -612,6 +612,9 @@ export class MerchCard extends LitElement {
                 !e.shiftKey &&
                 document.activeElement === lastFocusableElement
             ) {
+                const merchCards = document.querySelectorAll(MERCH_CARD_NODE_NAME);
+                const lastMerchCard = merchCards[merchCards.length - 1];
+                if (this === lastMerchCard) return;
                 e.preventDefault();
                 e.stopImmediatePropagation();
             } else if (
```
[the remark this answers is not in the document]
Comment (npeltier) [reply web-components/src/merch-card.js:620]: i'm not sure tbh :-| you should ask PMs

**Review by npeltier — APPROVED** [review]

**Review by rahulgupta999 — APPROVED** [review]

**Comment by 3ch023** [comment]
@rohitsahu closing this one since it was already merged in Milo iirc. If not, please reopen the same pr in milo.

**Pull request closed without merging**